test/controllers/company.number.controller.spec.unit.ts

```diff
@@ -0,0 +1,86 @@
+import app from "../../src/app";
+import * as request from "supertest";
+import {loadSession} from "../../src/services/redis.service";
+import {loadMockSession} from "../mock.utils";
+import {COOKIE_NAME} from "../../src/properties";
+import * as pageURLs from "../../src/model/page.urls"
+import Session from "../../src/session/session";
+
+jest.mock("../../src/services/redis.service");
+
+const NO_COMPANY_NUMBER_SUPPLIED = "No company number supplied";
+const INVALID_COMPANY_NUMBER = "Invalid company number";
+const COMPANY_NUMBER_TOO_LONG = "Company number too long";
+
+describe("company number validation tests", () => {
+
+   // This will be needed when the routes are secured
```

Comment:
Personal preference maybe, but I like to use a tag (something like TODO) in the comment, just so this can be searched for later on to find any blocks of code like this that need to be revisited/reincluded.